run.py

```diff
@@ -13,3 +13,4 @@ def add_csp_headers(response):
 if __name__ == '__main__':
     # vulnerability: Security Misconfiguration
     app.run(debug=True)
```

Comment:
[comment]: <> (FrogbotReviewComment)


## 📦🔍 Contextual Analysis CVE Vulnerability
<div align='center'>

| Severity                | Impacted Dependency                  | Finding                  | CVE                  |
| :---------------------: | :-----------------------------------: | :-----------------------------------: | :-----------------------------------: |
| ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableCriticalSeverity.png)<br>Critical | werkzeug:1.0.1 | The vulnerable function flask.Flask.run is called | CVE-2022-29361 |

</div>

<details>
<summary> <b>Description</b> </summary>
<br>

The scanner checks whether the vulnerable `Development Server` of the `werkzeug` library is used by looking for calls to `werkzeug.serving.run_simple()`.

</details>

<details>
<summary> <b>CVE details</b> </summary>
<br>

Improper parsing of HTTP requests in Pallets Werkzeug v2.1.0 and below allows attackers to perform HTTP Request Smuggling using a crafted HTTP request with multiple requests included inside the body. NOTE: the vendor's position is that this behavior can only occur in unsupported configurations involving development mode and an HTTP server from outside the Werkzeug project

</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>